client/state/data-layer/http-data/getters.js

```diff
@@ -0,0 +1,38 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import makeJsonSchemaParser from 'lib/make-json-schema-parser';
+import { http as rawHttp } from 'state/http/actions';
+import { requestHttpData } from 'state/data-layer/http-data/common';
```

Comment:
What if the public API of the `http-data` module could be imported from the root `state/data-layer/http-data` path? Leave the submodules for the internal stuff, i.e., the enhancer, reducer, effect handler etc.